ui/raidboss/data/triggers/o9s.js

```diff
@@ -170,8 +178,70 @@
         };
       },
     },
+    {
+      id: 'O9S Orbs Fiendish Phase Tracker',
+      regex: /14:317(C|D):Chaos starts using Fiendish Orbs/,
+      // condition: function(data) {
```

Comment:
Similar to my comments on the long/lat trigger, can you make a separate Phase Tracker trigger so that the alarmText for tanks picking up orbs can be its own trigger?

Reply:
Hey thanks for the thoughts yeah I can do that going forward as well and make these adjustment